### ISSUE TITLE
Something broke with Yeoman 1.0

### ISSUE THREAD

**Issue by mysterycommand** [pwmckenna/generator-travis-ci#5]
I haven't used this package in a while, so I'm not exactly sure when/where it happened, but with the current version of Yeoman … or rather the `yo` command line executable … the `yo travis-ci` process fails. Here's the output:

```
$ yo travis-ci

undefinedundefinedundefinedundefinedundefinedundefinedundefinedundefinedundefinedundefinedundefinedundefinedundefinedundefinedundefinedundefinedundefinedundefinedundefinedundefinedundefinedundefinedundefinedundefinedundefinedundefinedundefinedundefinedundefinedundefinedundefinedundefinedundefinedundefinedundefinedundefinedundefinedundefinedundefinedundefinedundefinedundefinedundefinedundefinedundefinedundefinedundefinedundefinedundefinedundefinedundefinedundefinedundefinedundefinedundefinedundefinedundefinedundefinedundefinedundefinedundefinedundefinedundefinedundefinedundefinedundefinedundefinedundefinedundefinedundefinedundefinedundefinedundefined


✔ Initialize GitHub Api
✔ Initialize Travis-ci Api
✔ Query Repository Information

GitHub Username (mysterycommand) 
GitHub Password **********


✔ Login to GitHub Api
✔ Ensure GitHub Travis App Authorized
✔ Generate GitHub OAuth Token
✔ Login to Travis-ci Api
undefinedEnsure Travis Repository Hook Set
undefinedReadme Build Status Image
undefinedRevoke GitHub OAuth Token
```

I'll see what else I can figure out.

**Comment by mysterycommand** [comment]
Wait … maybe I was just doing it wrong? Running `yo travis-ci:gh-pages` did work. Though I still got a bunch of `undefined`s as output. Let me know if I can help debug.

**Comment by pwmckenna** [comment]
i'll take a look at it this weekend and keep you posted. sounds pretty bad :(

**Comment by pwmckenna** [comment]
sorry for the delay...yeoman-generator switched from using colors to chalk, and unfortunately there was a hidden dependency on colors. fixed and released as 0.6.4. https://github.com/pwmckenna/generator-travis-ci/commit/af4b4609cb847c4a8601e5cdac1bf771a34be84e
